compiler/resolution/functionResolution.cpp

```diff
@@ -1770,8 +1962,13 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         // Otherwise, do the substitution now.
         subSymbol(workingFn->body, formal, var);
       }
+     } else {
+      // !needTupleInBody
+      substituteVarargTupleRefs(workingFn->body, n, formal, varargFormals);
+     }
 
```

Comment:
On May 11, 2016, at 1:35 PM, Vassily Litvinov <notifications@github.com<mailto:notifications@github.com>> wrote:


In compiler/resolution/functionResolution.cpp<https://github.com/chapel-lang/chapel/pull/3838#discussion_r62921818>:

> @@ -1770,8 +1962,13 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
>          // Otherwise, do the substitution now.
>          subSymbol(workingFn->body, formal, var);
>        }
> +     } else {
> +      // !needTupleInBody
> +      substituteVarargTupleRefs(workingFn->body, n, formal, varargFormals);
> +     }
>


I avoid messing revision history on existing code whenever I reasonably can.

1) I see the benefit but it’s more important to have consistently formatted code.
It would be silly to manage changes to the code base to favor preserving history.


2) This isn’t going to make the history any harder to inspect.



Given that the only issue here is indenting by 1 space instead of 2 spaces, do you insist on doing so?


Yes



—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/3838/files/20b76cba1018c724ee2d739174a99cfcfbb22a17#r62921818>